cpg-concepts/src/main/kotlin/de/fraunhofer/aisec/cpg/graph/concepts/ontology/InstallUpdateOperation.kt

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) 2025, Fraunhofer AISEC. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ *                    $$$$$$\  $$$$$$$\   $$$$$$\
+ *                   $$  __$$\ $$  __$$\ $$  __$$\
+ *                   $$ /  \__|$$ |  $$ |$$ /  \__|
+ *                   $$ |      $$$$$$$  |$$ |$$$$\
+ *                   $$ |      $$  ____/ $$ |\_$$ |
+ *                   $$ |  $$\ $$ |      $$ |  $$ |
+ *                   \$$$$$   |$$ |      \$$$$$   |
+ *                    \______/ \__|       \______/
+ *
+ */
+package de.fraunhofer.aisec.cpg.graph.concepts.ontology
+
+import de.fraunhofer.aisec.cpg.graph.Node
+import de.fraunhofer.aisec.cpg.graph.concepts.Operation
+import java.util.Objects
+import kotlin.Any
+import kotlin.Boolean
+import kotlin.Int
+
+public open class InstallUpdateOperation(
+    public val automaticUpdates: AutomaticUpdates?,
+    public val linkedConcept: AutomaticUpdates,
```

Comment:
Declaration as field seems redundant as we pass it to the super class, where it's stored. Moreover, modifier is used inconsistently parameters passed as arguments to the super class do not have modifier in almost all cases. Please remove `public val`.